### DISCUSSION TITLE
Verification of domain

### DISCUSSION THREAD

**Discussion by zasulskii** [WalletConnect/walletconnect-monorepo#3673] [Developer Support]
Can't make a verification by domain onyx.org
I wrote a TXT domain record and /.well-known/walletconnect.txt but still I'm getting this error message while verifying domain
![image](https://github.com/WalletConnect/WalletConnectSwiftV2/assets/50208720/debe01c5-7a44-4ed5-b62f-5356a15dec2f)

![image](https://github.com/WalletConnect/WalletConnectSwiftV2/assets/50208720/1339a803-b889-49ed-9aee-08c99edef9ea)

**Comment by glitch-txs** [comment]
Hi, are you still experiencing this issue?

**Comment by haiifeng** [comment]
+1
